client/src/charts/wrapped_nivo/WrappedNivoCircleProportion/WrappedNivoCircleProportion.js

```diff
@@ -119,101 +38,96 @@ export class WrappedNivoCircleProportion extends React.Component {
     const color_scale = scaleOrdinal().range(newIBCategoryColors);
     const value_formatter = get_formatter(is_money, formatter, true, false);
 
-    const graph_data = {
-      id: parent_name,
-      name: parent_name,
-      // ... nivo bubble will roll child values back up and add them to the parent value for use in the graph
-      // so we need to remove the inner portion from the total here
-      value: parent_value - child_value,
-      color: color_scale(parent_name),
-      children: child_value
-        ? [
-            {
-              id: child_name,
-              name: child_name,
-              value: child_value,
-              color: color_scale(child_name),
-            },
-          ]
-        : 0,
+    const Circles = () => {
+      // arbitrary parent values
+      const minX = 350;
+      const minY = 350;
+      const parent_radius = minX - 330;
+      const parent_cx = 175;
+      const parent_cy = 25;
+
+      // child circle calculations
+      const child_percent = child_value / parent_value;
+
+      // precaution against smaller child_percent values
+      const true_child_radius = parent_radius * child_percent;
+      const default_child_radius = parent_radius * 0.1;
+      const child_radius =
+        true_child_radius > default_child_radius
+          ? true_child_radius
+          : default_child_radius;
```

Comment:
Check the logic from the old component, it was correct. Want something like
```
const child_radius = _.max([
  parent_radius * Math.sqrt(child_percent), // do the math, make the actual area proportional,
  MIN_NODE_RADIUS,
]);
```

The square root is important! We want the areas of the circles to be proportional, which _isn't_ the same as using proportional radius (since the area itself is proportional to r^2). 

In the old case, `MIN_NODE_RADIUS` was `2` and was picked to be, I think, the smallest radius value that gave a visible inner circle (although this `2` could also be started relative to the svg viewbox dimensions`. We need to be very  careful with this sort of "fudging" of visualizations, misrepresenting the data is the last thing we want. Trying to pick the smallest possible visible radius for the lower limit is preferable as it's too small to actually communicate a proportion anyway, it just communicates that the inner value is not zero.

Reply:
Done! I think the formula provided is correct.